apps/web/src/app/agents/[agentId]/overview/page.tsx

```diff
@@ -0,0 +1,30 @@
+import { notFound } from "next/navigation";
```

Comment:
this page should be moved to just apps/web/src/app/agents/[agentId]/page.tsx. no 'overview'. Adjust links accordingly.